src/test/java/com/kodekonveyor/work_request/revoke/RevokeWorkRequestControllerTest.java

```
@@ -0,0 +1,35 @@
+package com.kodekonveyor.work_request.revoke;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.junit.runner.RunWith;
+import org.mockito.Mockito;
+import org.mockito.junit.MockitoJUnitRunner;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.mockito.junit.jupiter.MockitoSettings;
+import org.mockito.quality.Strictness;
+
+import com.kodekonveyor.annotations.TestedBehaviour;
+import com.kodekonveyor.annotations.TestedService;
+
+@ExtendWith(MockitoExtension.class)
+@MockitoSettings(strictness = Strictness.LENIENT)
+@RunWith(MockitoJUnitRunner.class)
+@TestedBehaviour("Save the new/modified entities")
+@TestedService("RevokeWorkRequestController")
+public class RevokeWorkRequestControllerTest
```

Negative test cases are missing.

---

I think we should throw an exception when no element has been deleted (the workRequestRepository.deleteById() method returns 0).
@magwas what is your opinion. Shall we throw an error or do nothing quietly when the id of the deleting element not found?

---

I think throwing a ValidationEception (which is handled by RestResponseEntityExceptionHandler ) is the way to go.